docs/data-access/icgc-25k-data.md

```diff
@@ -0,0 +1,100 @@
+---
+id: icgc-25k-data
+title: ICGC 25K Data Access
+platform_key: DOCS_ICGC_25K
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The ICGC Data Portal was retired in June 2024.
+
+The [Data Portal](https://pubmed.ncbi.nlm.nih.gov/21930502/) served as a hub and reponsitory, a culmination of the collaborative effort between International Cancer Genome Consortium and multiple partners from various cancer projects (including [TCGA](https://www.cancer.gov/ccg/research/genome-sequencing/tcga) and [Sanger Cancer genome project](https://www.sanger.ac.uk/group/cancer-genome-project/)).
+
+The aim was to analyze multiple cancer types and share open access data such as simple somatic mutations, copy number alterations, structural rearrangements, gene expression, microRNAs, DNA methylation and exon junctions.
+
+### Relocated ICGC 25K Data
+
+Although the interactive data portal was shut down, data from the project remains available. The final release data, as well as the PCAWG project data are now hosted by ICGC ARGO. This data is [available to authorized users via an SFTP](#accessing-icgc-25k-release-data).
```

Comment:
Still find the mention of ICGC ARGO confusing. 

"The final release data, as well as the PCAWG project data remain [available to authorized users via an SFTP server]"

Reply:
How about combining the two sentences to be more explicit:

```md
The final release data, as well as the PCAWG project data are now available to authorized users through an [SFTP server](#accessing-icgc-25k-release-data) hosted by ICGC ARGO.
```